### ISSUE TITLE
CDN?

### ISSUE THREAD

**Issue by dheera** [Polymer/polymer#2714]
Is there a CDN for Polymer and Material Elements that we can use? It's somewhat tedious to and repetitive to have to download a copy of everything into every single project, and potentially bandwidth-inefficient. Thanks!

**Comment by abdonrd** [comment]
http://polygit.org :blush:

**Comment by dheera** [comment]
Thanks!

**Comment by dheera** [comment]
@abdonrd I tried using it for a little while and it seems to be rather slow, resulting in delays of several seconds. I see from the headers that it is using Cloudflare which seems to me that it *should* be fast. It may be that the "max-age=3600" header is causing constant reloads of the entire Polymer code every time. On my own server I set the max-age to several weeks, and include the version number in the URL to permit updating via changing the URL.

I hope eventually Google themselves will use their own CDN for this as they'd probably be able to build optimizations into Chrome itself to maintain the latest copies of commonly-used libraries like jQuery, Polymer, Angular, and so on. Another potential solution for Google would be to having some kind of checksum="..." field inside the <script> tag where if the same file was already obtained from another source or server, it is not re-downloaded.
